src/server/systems/NetworkingServer.cpp

```diff
@@ -168,23 +174,41 @@ void NetworkingServer::handleUnreliableCommand(sf::Socket::Status status, sf::Pa
             break;
         }
         default: {
-            std::cout << "Unknown command code sent: " << unreliable_command.command << std::endl;
+            std::cerr << "Unknown command code sent: " << unreliable_command.command << std::endl;
```

Comment:
currently cerr clobbers terminal UI, redirecting stderr to a file should solve this issue, but Im brainstorming ways to output cerr to the termbox terminal UI (probably means using a signleton)

Reply:
but im worried about thread safety issues with the singleton